### DISCUSSION TITLE
openOCD Debugging

### DISCUSSION THREAD

**Discussion by nuhaildam** [Mair/esp32-course#109] [Show and tell]
Hi Mair,
I'm having segger j-link with me for openocd debugging, but I'm unable to get it to work.
In the esp-32 course you've used FT2232 board, but I'm only having segger j-link and I couldn't find any proper tutorial for that online.
Can you please guide me how to use segger j-link for openocd debugging? 
I'm using ESP32 DEVKITV1(ESP32-WROOM-32D) board.

**Comment by Mair** [comment]
It should work so long as you wire it correctly and have all the other steps as described.
Please see https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/jtag-debugging/index.html

Hope that helps. If not let's see your errors that you are getting
